src/components/tabs/tabs.component.js

```diff
@@ -356,6 +358,10 @@ Tabs.propTypes = {
     "no right side",
     "no sides",
   ]),
+  /** sets width to the tab headers. Can be any valid CSS string.
+   * The headerWidth prop works only for `position="left"`
```

Comment:
Is it worth maybe having a custom prop validation here now that this prop works only when the `position="left"`